Build/Tactical/OppList.cc

```diff
@@ -150,14 +150,6 @@ static const INT8 gbLookDistance[8][8] =
 /* NORTHWEST */ { ANGLE,          SIDE,  SBEHIND,     BEHIND,  SBEHIND,       SIDE,    ANGLE,  STRAIGHT }
 };
 
-static const INT8 gbSmellStrength[3] =
```

Comment:
Looking at it, the constants are defined elsewhere, eg. NORMAL_HUMAN_SMELL_STRENGTH and COW_SMELL_STRENGTH. It can stay then, I don't see any +/-2 uses.